x/coinswap/internal/types/querier.go

```diff
@@ -8,3 +8,15 @@ const (
 	ParamFee         = "fee"
 	ParamNativeDenom = "nativeDenom"
 )
+
+// defines the params for the following queries:
+// - 'custom/coinswap/liquidity'
```

Comment:
It doesn't necessarily need to match the path specified in registerQueryRoutes, because I am defining the params for the queries. I agree that the param is a part of the query, but it's being defined specifically for that query. For instance: Check querier.go of the slashing module.